frontend/containers/forms/project-gallery/project-gallery-image/component.tsx

```diff
@@ -18,18 +22,29 @@ export const ProjectGalleryImage = <FormValues extends FieldValues>({
   registerOptions,
   invalid: invalidProp = false,
   defaultSelected = false,
+  onDeleteImage,
   ...rest
 }: ProjectGalleryImageProps<FormValues>) => {
   const [invalid, setInvalid] = useState<boolean>(invalidProp);
   const { isFocusVisible, focusProps } = useFocusRing();
   const { id, title, src } = image;
+  const { formatMessage } = useIntl();
 
   useEffect(() => {
     setInvalid(invalidProp);
   }, [invalidProp]);
 
   return (
-    <div className="relative rounded">
+    <div className="relative rounded group">
+      <Button
+        theme="primary-white"
+        className="absolute right-0 z-10 justify-center w-6 h-6 px-0 py-0 mx-2 my-2 overflow-hidden text-red-600 transition-opacity ease-in opacity-0 group-hover:opacity-100 group-hover:text-red-600 focus-visible:ring-2 focus-visible:ring-green-dark focus-visible:opacity-100 focus-visible:ring-offset-2"
```

Comment:
The focus ring is not correctly shown. The `<Button />` has a focus ring by default so you don't have to define it here:
```suggestion
        className="absolute right-0 z-10 justify-center w-6 h-6 px-0 py-0 mx-2 my-2 overflow-hidden text-red-600 transition-opacity ease-in opacity-0 group-hover:opacity-100 group-hover:text-red-600 focus-visible:opacity-100"
```